Generate docs (#2356)
kyma-project/cli: docs/user/gen-docs/01-170-kyma_alpha_reference-instance.md

# kyma alpha reference-instance

Adds an instance reference to a shared service instance.

## Synopsis

> Use this command to add an instance reference to a shared service instance on the Kyma cluster.

Author: ```suggestion
Use this command to add an instance reference to a shared service instance in the Kyma cluster.
```

# kyma alpha reference-instance

> Add an instance reference to a shared service instance

Reviewer: ```suggestion
Adds an instance reference to a shared service instance.
```

## Synopsis

> Use this command to add an instance reference to a shared service instance on the SAP Kyma platform

Reviewer: ```suggestion
Use this command to add an instance reference to a shared service instance on SAP BTP.
```

Author: This command is adding things the kyma cluster so I would suggest adding the `on the Kyma cluster` instead of `on SAP BTP` at the end of the description

```bash
kyma alpha reference-instance [flags]
```

## Flags

```bash
      --btp-secret-name string   name of the BTP secret containing credentials to another subaccount Service Manager:
                                 https://github.com/SAP/sap-btp-service-operator/blob/main/README.md#working-with-multiple-subaccounts
      --instance-id string       ID of the instance
      --label-selector strings   Label selector for filtering instances
      --name-selector string     Instance name selector for filtering instances
      --namespace string         Namespace of the reference instance (default "default")
      --offering-name string     Offering name
      --plan-selector string     Plan name selector for filtering instances
      --reference-name string    Name of the reference
```

## Flags inherited from parent commands

```bash
  -h, --help                    Help for the command
      --kubeconfig string       Path to the Kyma kubeconfig file
      --show-extensions-error   Print possible error when fetching extensions failed
```

## See also

> * [kyma alpha](kyma_alpha.md) - Groups command prototypes the API for which may still change

Reviewer: ```suggestion
* [kyma alpha](kyma_alpha.md) - Groups command prototypes for which the API may still change.
```
